.env.example

```
@@ -1,51 +1,74 @@
+# ----------------------------------------------------------------------------
+# STIX MΛGIC environment template
+# Copy to .env for local development:
+#   cp .env.example .env
+# ----------------------------------------------------------------------------
+
 NODE_ENV=development
+
+# Local service ports
 WEB_PORT=3000
 API_PORT=4000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_PORT key should go before the WEB_PORT key

(UnorderedKey)